engine/Mono.Cecil.Shared/Mono.Cecil/ModuleKind.cs

```diff
@@ -29,6 +85,7 @@ public enum TargetArchitecture {
 		ARMv7 = 0x01c4,
 		ARM64 = 0xaa64
 	}
+	/* End AGPL */
```

Comment:
This line is not positioned right as there are no changes to TargetArchitecture.